### PR TITLE
Rename StreamTexture* to CompressedTexture*

### PR DESCRIPTION
**Pull request by reduz**
* It is not and will not be used for streaming.
* Streaming will be implemented in 4.1 and it will work different.
* It makes more sense to be called CompressedTexture since it imports and compresses texture files.

### CODE REVIEW

**Comment by akien-mga** [comment]
Another option we discussed on chat was `ImportedTexture*`.

I'm fine with both, if anyone feels strongly in favour of one or the other, do comment. @godotengine/rendering @godotengine/import

**Comment by reduz** [comment]
@akien-mga We will likely have other flavors of imported texture when we add a proper streamable texture, so I think it may just be better to leave it as Compressed, which is mostly the aim of this one.

**Comment by Calinou** [comment]
I think ImportedTexture can be confusing, especially since other imported resources don't have Imported in their name.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

File: doc/classes/CompressedTexture2D.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	</methods>
 	<members>
 		<member name="load_path" type="String" setter="load" getter="get_load_path" default="&quot;&quot;">
-			The StreamTexture's file path to a [code].stex[/code] file.
+			The CompressedTexture's file path to a [code].stex[/code] file.
```
Comment (TechnoPorg) [inline doc/classes/CompressedTexture2D.xml:22]: I think this should be changed to `.ctex`.

Comment (Calinou) [reply doc/classes/CompressedTexture2D.xml:22]: Fixed by https://github.com/godotengine/godot/pull/58965.

**Pull request merged**